src/app/map/[id]/view/[viewIdOrHost]/publish/components/PublicMapTopBarMobile.tsx

```diff
@@ -22,134 +18,20 @@ export default function PublicMapTopBarMobile() {
-      {isOverlayOpen && (
-        <div className="fixed inset-0 z-50 bg-black/50 flex items-start justify-center pt-16">
-          <div className="bg-white rounded-lg shadow-2xl w-full max-w-md mx-4 max-h-[80vh] overflow-y-auto">
-            {/* Header */}
-            <div className="flex items-center justify-between p-4 border-b border-white/20">
-              <div className="flex items-center gap-2">
-                <MapIcon
-                  className="w-8 h-8 shrink-0"
-                  style={{
-                    fill: "white",
-                    stroke: activeColourScheme.primary,
-                  }}
-                />
-                <EditablePublicMapProperty
-                  property="name"
-                  placeholder="Map name"
-                >
-                  <h1 className="text-xl font-medium leading-tight text-balance tracking-tight">
-                    {publicMap.name}
-                  </h1>
-                </EditablePublicMapProperty>
-              </div>
-              <button
-                onClick={() => setIsOverlayOpen(false)}
-                className="p-1 hover:bg-white/20 rounded transition-colors"
-              >
-                <X
-                  className="w-5 h-5"
-                  style={{ color: activeColourScheme.primary }}
-                />
-              </button>
-            </div>
-
-            {/* Content */}
-            <div className="p-4 flex flex-col gap-4">
-              {/* Description */}
-              <div className="flex flex-col gap-1">
-                <EditablePublicMapProperty
-                  property="description"
-                  placeholder="Map description"
-                >
-                  <p className="text-sm">
-                    {publicMap.description || (
-                      <span className="text-sm text-neutral-500 italic">
-                        Add a description to your map
-                      </span>
-                    )}
-                  </p>
-                </EditablePublicMapProperty>
-              </div>
-
-              {/* Link */}
-              <div className="flex flex-col gap-1">
-                <EditablePublicMapProperty
-                  property="descriptionLink"
-                  placeholder="submissions@example.com"
-                >
-                  {publicMap.descriptionLink && (
-                    <a
-                      className="underline text-sm"
-                      style={{
-                        color: activeColourScheme.primary,
-                      }}
-                      href={publicMap.descriptionLink}
-                      target="_blank"
-                      onClick={(e) => editable && e.preventDefault()}
-                    >
-                      {publicMap.descriptionLink}
-                    </a>
-                  )}
-                </EditablePublicMapProperty>
-              </div>
-            </div>
-          </div>
-        </div>
-      )}
-    </>
```

Comment:
I removed the editor stuff from here cause I don't think the editor view will work on mobile? And it was really complicatedddd